core/templates/pages/about-page/about-page.scripts.ts

```diff
@@ -21,5 +21,5 @@
 require('pages/about-page/about-page.module.ts');
 require('App.ts');
 
-require('pages/about-page/about-page.component.ts');
+// require('pages/about-page/about-page.controller.ts');
```

Comment:
**Do not commit commented-out code.** Please proofread your PR thoroughly before sending it for review.

Reply:
Removed this line of code. Not required anymore!